backend/config/corpora-api.yml

```diff
@@ -499,6 +499,26 @@ paths:
         "403":
           $ref: "#/components/responses/403"
 
+  /dp/v1/genesets/{geneset_uuid}:
+    delete:
+      tags:
+        - genesets
+      summary: Delete gene sets from a collection
```

Comment:
nit: singular instead of plural, since this endpoint only supports deleting one gene set at a time?

Reply:
Since this fully deletes the geneset maybe the summary should be Delete a geneset and the description can say delete a gene set removing it from a collection and any data sets it was linked to.